docs/content/preview/yugabyte-platform/administer-yugabyte-platform/high-availability.md

```diff
@@ -134,6 +134,12 @@ To confirm communication between the active and standby, you can do the followin
 
     ![Verify Prometheus](/images/yp/high-availability/ha-prometheus.png)
 
+    {{<note title="Metrics availability on the standby">}}
+Metrics on the standby are only available from the time the standby was activated. When activated, the standby begins collecting metrics, and no historical metrics are copied from the active instance at that time.
```

Comment:
minor: this could be taken to mean the standby collects its own metrics from the db ndoes, whereas it only "tails" the active instance's prometheus data. Maybe we could say "begins collecting metrics from the active instance"